### ISSUE TITLE
Allow `%file()%` command to use `../` to access parent folders of a file

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#237]
The `%file()%` command currently supports accessing files from a location relative to where the included HTML file / picture is stored.

The same should apply to the `%style()%` command.

Sometimes we need to include file content of a file that is in a sibling or parent folder, so for this we will need to be able to use the sequence ../ to change to the parent directory and access its contents (or subfolders therein). This currently doesn';t work.

Examples:

```
%file(../../history/qf-5.16-de.html)%
%file(../faq.html)%
%style(../styles/quickfolders-reminder.css)%
```

**Comment by RealRaven2000** [comment]
Test version below. I needed the feature myself, I implemented it. Also updated News Splash accordingly. (I might add a new example to the variables tab as well)

[smartTemplate-fx-3.17pre28.zip](https://github.com/RealRaven2000/SmartTemplates/files/11322392/smartTemplate-fx-3.17pre28.zip)

To install, download the zip file and then drag it into Thunderbird Add-ons Manager (do not extract contents, because you cannot install it that way).

**Comment by RealRaven2000** [comment]
Implemented in v3.17 released 04/May/2023
